src/randoop/NoExceptionCheck.java

```diff
@@ -58,7 +62,7 @@ public String toCodeStringPostStatement() {
 
   @Override
   public String toCodeStringPreStatement() {
-    return "";
+    return "// contract failure: this statement threw " + exceptionName + Globals.lineSep;
```

Comment:
Wouldn't it be a contract failure to throw *any* exception?
Or is this just about some specific exception and other exceptions are OK?